app/Dockerfile

```diff
@@ -20,11 +20,11 @@ RUN \
   apt-get update \
   && apt-get install --no-install-recommends -yqq \
   netcat=1.10-46 \
-  gcc=4:10.2.1-1 \
-  graphviz=2.42.2-5
```

Comment:
To fix the issue we were encountering when running ./scripts/buildrun.sh, removing this line was just a local quick fix. Instead it may be better to specify graphviz=2.42.* so that the newest patch is installed instead (i.e. graphviz_2.42.2-5+deb11u1 instead of graphviz_2.42.2-5).